xonsh/built_ins.py

```diff
@@ -714,6 +715,7 @@ def load_builtins(execer=None, config=None, login=False, ctx=None):
     builtins.__xonsh_ensure_list_of_strs__ = ensure_list_of_strs
     builtins.__xonsh_list_of_strs_or_callables__ = list_of_strs_or_callables
     builtins.__xonsh_completers__ = xonsh.completers.init.default_completers()
+    builtins.__xonsh_events__ = Events()
```

Comment:
Yeah, events is not going to be able to live here forever.  Also, I agree and think that since this is a public interface, like aliases, it should be called `events`